content/docs/react-without-jsx.md

```diff
@@ -53,7 +53,6 @@ ReactDOM.render(
 );
 ```
 
-If you use this shorthand form for `React.createElement`, it can be almost as convenient to use React without JSX.
-
-Alternatively, you can refer to community projects such as [`react-hyperscript`](https://github.com/mlmorg/react-hyperscript) and [`hyperscript-helpers`](https://github.com/ohanhi/hyperscript-helpers) which offer a terser syntax.
+यदि आप `React.createElement` के लिए इस शॉर्टहैंड फॉर्म का उपयोग करते हैं, तो JSX के बिना React का उपयोग करना लगभग सुविधाजनक हो सकता है।
 
+वैकल्पिक रूप से, आप [`रीएक्ट-हाइपरस्क्रिप्ट`](https://github.com/mlmorg/react-hyperscript) और [`हाइपरस्क्रिप्ट-हेल्पर्स`](https://github.com/ohanhi/hyperscript-helpers) जैसी सामुदायिक परियोजनाओं को भी देख सकते हैं, जो एक संक्षिप्त वाक्य रचना प्रदान करती हैं।
```

Comment:
`परियोजनाओं` => `प्रोजेक्ट्स`

Reply:
`वैकल्पिक रूप से` => `eske alawa`

how about this?

Reply:
Add a line break after line 58

Reply:
why do we need a line break after line 58? I thought that was there by mistake.
`इसके अलावा` doesn't seems good, instead of that we can change it to `इसके अतिरिक्त`.

Reply:
I suggested that to simplify. Atirikt seems a bit complex word. We want to keep it as simple as possible